### PR TITLE
Fix 13 release :)

### PR DESCRIPTION
**Pull request by MariusBluem**
This needs to be 13 - otherwise the docs URL will be generated wrong. The other one stays 13.0.0 for the download link :) Please review and deploy!!

### CODE REVIEW

File: strings.php [hunk 1]
```diff
@@ -2,7 +2,7 @@
 // Strings used through nextcloud.com
 
 /* VERSIONS */
-$VERSIONS_SERVER_MAJOR_STABLE = '13.0.0';
+$VERSIONS_SERVER_MAJOR_STABLE = '13';
 $VERSIONS_SERVER_FULL_STABLE = '13.0.0';
 $VERSIONS_SERVER_MAJOR_DEVELOPMENT = '13.0.0';
```
Comment (MariusBluem) [inline strings.php]: This should be 14 or empty now...

File: strings.php [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 $VERSIONS_SERVER_MAJOR_DEV_DOCS = '13'; // Used in dev docs links
```
Comment (MariusBluem) [inline strings.php]: Same here...

**Comment by MariusBluem** [comment]
And please see my comments above... am I right?

File: strings.php (hunk 2 above)
Comment (jospoortvliet) [reply strings.php]: you are correct, this is wrong, our mistake.

Comment (MorrisJobke) [reply strings.php]: Please keep it - we don't have 14 yet available on the docs server.

**Pull request merged**